components/renku_data_services/session/core.py

```diff
@@ -118,9 +119,11 @@ def validate_session_launcher_patch(
         resource_class_id = RESET
     else:
         resource_class_id = patch.resource_class_id
+    disk_storage = RESET if "disk_storage" in data_dict and data_dict["disk_storage"] is None else patch.disk_storage
```

Comment:
issue: What if we don't want to change the disk storage in the PATCH? If we don't set it, the value will be `None` and it causes the disk storage to be reset (in the followup code).

Reply:
A value of `None` does NOT reset the value during the patch operation.